packages/p2p/src/pages/orders/chat/chat-wrapper.jsx

```diff
@@ -19,7 +19,8 @@ const ChatWrapper = observer(({ children, is_modal_open }) => {
             pageHeaderReturnFn={() => {
                 sendbird_store.setShouldShowChatModal(false);
                 sendbird_store.setShouldShowChatOnOrders(false);
-                order_store.hideDetails(true);
+                order_store.should_navigate_to_order_details && order_store.hideDetails(true);
+                order_store.setShouldNavigateToOrderDetails(false);
```

Comment:
I tried to use history.goBack function but this won't be effective because it gets rid of the order id from the url even though it goes back to the details page. This can be resolved in v2 because everything is routable and will be in the url.